cmds/rm/rm.go

```diff
@@ -51,6 +54,10 @@ func rm(files []string) error {
 		f = os.RemoveAll
 	}
 
+	if flags.f {
+		flags.i = false
```

Comment:
This is not what rm does on my machine. `rm -fi` still asks me to approve removal.

Reply:
True story, but `rm -if` doesn't :)

Which makes me consider this as unwanted implementation detail in how are arguments being parsed. While it really doesn't make sense to call `rm` with both `-i` and `-f` I find it more logical for a *force* flag to be with higher priority.

Reply:
I would agree.
On my sytem `rm -fi` does indeed prompt me to approve removals but still complains about missing files which leads me to believe the `f` argument is being ignored when followed by `i`when I would logically expect the opposite.
This seems to be more of an implementation issue rather than intended functionality:
```
$ rm --version
rm (GNU coreutils) 8.25
(...)
$ sudo touch 1
(...)
$ touch 2 3 4
$ rm -fifiiiii 1 2 3 4 
rm: remove write-protected regular empty file '1'? y
rm: cannot remove '1': Operation not permitted
rm: remove regular empty file '2'? y
rm: remove regular empty file '3'? y
rm: remove regular empty file '4'? y
$ touch 2 3 4
$ rm -fifiiiiif 1 2 3 4 
rm: cannot remove '1': Operation not permitted
$
```